### PR TITLE
Adds support for WBTC-B and WBTC-C maker vaults

### LINKED ISSUE #3952: Support new Maker Vault types
## Abstract

```
Detected vault with collateral_type WBTC-C. That is not yet supported by rotki. Skipping...
```

## Motivation

Would want this vault type to be supported by rotki, older WBTC types like e.g. WBTC-A are supported. Skimming through https://vote.makerdao.com/executive, it looks like this was the only new vault type since August 2021, more at https://vote.makerdao.com/executive/template-executive-vote-onboarding-new-vault-types-parameter-changes-and-core-unit-budget-distributions-november-26-2021?network=mainnet#proposal-detail

## Specification

Should be clear I think

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #3952

Got the addresses from https://github.com/makerdao/pymaker/blob/master/config/mainnet-addresses.json#L78

### CODE REVIEW

**Review by LefterisJP — COMMENTED** [review]
There is more vaults that are missing. I guess you can ignore the LP tokens (for now) but for example I see Matic which we don't support yet.

Add all missing ones as seen here: https://oasis.app/vaults/list

File: rotkehlchen/constants/ethereum.py [hunk 1]
```diff
@@ -111,6 +113,8 @@ def abi(name: str) -> List[Dict[str, Any]]:
 MAKERDAO_UNI_A_JOIN = EthereumConstants().contract('MAKERDAO_UNI_A_JOIN')
 MAKERDAO_RENBTC_A_JOIN = EthereumConstants().contract('MAKERDAO_RENBTC_A_JOIN')
 MAKERDAO_AAVE_A_JOIN = EthereumConstants().contract('MAKERDAO_AAVE_A_JOIN')
+MAKERDAO_WSTETH_A_JOIN = EthereumConstants().contract('MAKERDAO_WSTETH_A_JOIN')
```
Comment (LefterisJP) [inline rotkehlchen/constants/ethereum.py:116]: You seem to be adding this in the constants without actually adding support for this. Please either also support it and add to changelog or completely remove from here and from contracts json.

Comment (yabirgb) [reply rotkehlchen/constants/ethereum.py:116]: Was there for later but okey, removing it

**Comment by LefterisJP** [comment]
> Closes #3952
> 
> Got the addresses from https://github.com/makerdao/pymaker/blob/master/config/mainnet-addresses.json#L78

Another hint is to read the code. We have a link in the code that maker has to all their mainnet addresses that's kept up to date and even has all releases!

https://changelog.makerdao.com/releases/mainnet/active/contracts.json

(addresses are correct and check out)

**Comment by LefterisJP** [comment]
your force push somehow introduced conflicts? Or probably I did not notice this needede a rebase on bugfixes @yabirgb :sweat_smile:

**Review by LefterisJP — APPROVED** [review]
LGTM. Thanks @yabirgb

**Pull request merged**